tests/Console/ConfigurationResolverTest.php

```diff
@@ -961,6 +961,26 @@ public function provideResolveRulesCases()
                     '@A' => true,
                 ),
             ),
+            '@A - @C' => array(
```

Comment:
same for both.

`@X = [AA => true]`, `@Y = [AA => false]`

`@X,@Y => [AA => true] + [AA => false]` => `[AA => false]`
but
`@X,-@Y => [AA => true] - [AA => false]` => `[AA => true]`